server/examples.js

```diff
@@ -0,0 +1,209 @@
+import rp from 'request-promise';
+import Q from 'q';
+import mongoose from 'mongoose';
+import objectID from 'bson-objectid'
+import shortid from 'shortid';
+import User from './models/user';
+import Project from './models/project';
+
+const defaultHTML =
+`<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.4/p5.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.4/addons/p5.dom.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.4/addons/p5.sound.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <script src="sketch.js"></script>
+  </body>
+</html>
+`;
+
+const defaultCSS =
+`html, body {
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+}
+`;
+
+const a = objectID().toHexString();
+const b = objectID().toHexString();
+const c = objectID().toHexString();
+const r = objectID().toHexString();
+
+const client_id = process.env.CLIENT_ID;
+const client_secret = process.env.CLIENT_SECRET;
+
+const headers = {'User-Agent': 'p5js-web-editor/0.0.1'};
+
+mongoose.connect('mongodb://localhost:27017/p5js-web-editor');
```

Comment:
There's an environment variable called process.env.MONGO_URL, I think it would make sense to use that here so that if we decide to move the mongodb somewhere else, we only have to change the url in one place 😄